[WIP] Add initial decision making process to community handbook (#3940)
the-turing-way/the-turing-way: book/website/community-handbook/governance/gov-constitutional.md

(ch-gov-constitutional)=
# Consitutional level decision making

Decisions taken at the consitutional level affect multiple working groups and delivery groups and therefore a broad and diverse group of individual community members.
They will usually have long term implications for members of *The Turing Way* community.
Decisions taken at the consitutional level may require a prioritisation of activities, for example for the Community Management delivery groups(s).

Decisions at this level are made by the *The Turing Way*'s Steering Council.

## Membership

The Steering Council is made up of the chairs of the working groups and delivery groups.
This number is not limited.

### Electing a chair 

The Steering Council elects a chair from their membership.
The responsibilities of the chair are detailed in the <!-- MAKE INTO LINK-->role of a chair page in this chapter.

> The Chair of the Steering Council holds their role for a calendar year, starting in January and handing over their responsibilities in December.
> 
> The Chair joins as Chair Elect for one year, holds the role of Chair for one year, and remains on the Steering Council for one year as Past Chair.

Reviewer: ```suggestion
The Chair of the Steering Council holds their role for a calendar year, starting in January and handing over their responsibilities in December.
The Chair joins as Chair Elect for one year, holds the role of Chair for one year, and remains on the Steering Council for one year as Past Chair.
```

The wording feels a bit confusing here.

Does this mean a chair is elected one year before they take on the role?

> The Chair joins as Chair Elect for one year, holds the role of Chair for one year, and remains on the Steering Council for one year as Past Chair.

Reviewer: I like the idea that a chair is involved before and after their term. Hopefully it will help smooth the transitions and mean we can commit to long term goals spanning multiple terms.

In order to be elected the Chair must be an existing chair of a working group or delivery group.

> However, if they step down from those responsibilities before the end of their three year commitment to The Turing Way (as Chair Elect, Chair and Past Chair) they may remain in post.

Reviewer: We might want to think if this has implications on how working groups and delivery groups choose their chairs. If they change chairs often, people might miss the opportunity to be delivery group chair.

However, it is always possible for a Chair to step down if they choose to.

If the Chair Elect steps down from their role the Steering Committee will elect another member into the role.

If the Chair steps down from their role the Chair Elect will be asked to step into the role of Chair and will serve for the remainder of the Chair's term as well as their full year as Chair.

If the Chair Elect does not want to take on the Chair role earlier than planned then the Steering Committee will elect a Chair from their membership. 

The role of Chair can be shared by up to three Co-Chairs as described in the <!-- MAKE INTO LINK-->role of a chair page in this chapter.

## Ways of working

### Synchronous collaboration

The Steering Council will meet every 3 months online.

They will aim to host their meetings around 1 month before the community forum events to maintain a regular cadance of communication between the Steering Council and the broader community.

It is up to the Steering Council to decide how long each meeting should last.

It is the responsibility of the Chair to set the agenda for these meetings.

> They may choose to - for example - focus on particularly thorny topics for discussion, or to invite presentations on activity across the project to keep all Steering Council members informed.

Reviewer: ```suggestion
They may choose to, for example, focus on particularly thorny topics for discussion, or to invite presentations on activity across the project to keep all Steering Council members informed.
```

The Steering Council may also hold ad hoc meetings to discuss specific topics in order to facilitate empowered, transparent and inclusive decision making across the project. 

### Asynchronous collaboration

> Steering Council members will be added to a private slack channel in *The Turing Way* slack workspace.

Reviewer: ```suggestion
Steering Council members will be added to a private channel in *The Turing Way* Slack workspace.
```

They can use this space to share knowledge and to ask for input for ongoing discussions.

All requests for decisions by the Steering Council are documented as issues in the governance repository in *The Turing Way* GitHub organisation.

> Steering Council members are expected to monitor these issues and contribute on topics related to the interests and needs of their working group or delivery group members.

Reviewer: I like this focus on the "official" work being done in public rather than on Slack.

### Responding to requests for a decision

> Any individual member of The Turing Way community can request a decision from the Steering Council.

Reviewer: This feels like a difficult balance. You don't want the Steering Council to have too high a workload, or to be asked to rule on things that could be handled at a lower level. However, if requests had to come from the  maintenance level some things might be missed (if there is no suitable working group, for example).

They would make that request in the form of a GitHub issue in the goverance GitHub repository.
See the page in this chapter on <!-- FORMAT TO LINK --> balancing sensitive information disclosure with transparency for guidance on the appropriate amount of information to include in the public request.

Requests for decisions can also be brought by working groups and delivery groups, and by the Steering Council themselves.

On receiving the request for a decision, the Steering Council may:

1. Approve the recommended action if one is provided.
2. Develop and endorse a solution to the challenge if no recommended action was proposed.
3. Develop and endorse an alternative solution to the challenge if they do not agree with the proposed action.

> 4. Delegate the decision to a different team.

Reviewer: Maybe this is missing an option to reject the request?

I like this, feels like RFC.

* Not making a decision limits activity for community members and thus disempowers them.
* They can be transparent about which pieces of information they are not making publicly available and therefore foster trust by being careful with the information they hold.
* Making an inclusive decision means ensuring everyone has had a chance to feed into the process and have their perspective be heard.

In the event of the Steering Council not being able to reach consensus the final decision is delegated to the Chair of the Steering Council.

## Example decisions

Decisions that are appropriate to be made at the consitutional level include but are not limited to:

* Creating a new working group or delivery group.
* Restructuring the *The Turing Way* guides.
* Allocating paid staff time to working group, delivery group or individual contributor activities.
* Reviewing project governance processes.
* Providing a letter of support for an initiative on behalf of the *Turing Way* community.

The steering group can be asked to take 3 actions one is to note decisions that were made in the last quarter by a working group or delivery group. This goes under in form heading

> They can also be asked to request to give approval for a proposal that is usually one that's been developed across working groups and delivery groups and that they are in unanimous support for.

Reviewer: ```suggestion
They can also be asked to give approval for a proposal that is usually one that's been developed across working groups and delivery groups and that they are in unanimous support for.
```

### Hypothetical worked example: Collaborating on a funding application

Imagine that there is a funding call looking to support training for open science in India and to build international networks in open science.

The funder is based in India, the money can only be distributed to and managed by an organisation based in India, and the applicants are required to have someone based at an organisation outside of India to be part of the collaboration.

Indrani is a professor at Delhi University in India and Baptiste is a graduate student at Montpellier University in France.

> They have both contributed to The Turing Way as individual contributors at the community decision making level.

Reviewer: ```suggestion
They have both contributed to *The Turing Way* as individual contributors at the community decision making level.
```

> Their idea involves improving resources from The Turing Way and promoting them in their professional networks.

Reviewer: ```suggestion
Their idea involves improving resources in *The Turing Way* and promoting them in their professional networks.
```

They collaboratively develop their funding proposal with input from the Translation and Localisation working group.
Before submitting the grant (ideally) they bring a request to the Steering Council to create (if approved) a new delivery group.
They want to use this funding to strengthen interpersonal and professional connections across The Turing Way, invite new people into the community, and to improve the content for a broader audience.

This Delivery Group would be chaired by Indrani as principle investigator of the project.

> Baptiste would be a member, and Marc - a member of the Turing Way's Translation and Localisation working group - becomes a member of this delivery group in order to maintain strong connections between the T&L Working Group and this delivery group.

Reviewer: ```suggestion
Baptiste would be a member, and Marc — a member of *The Turing Way*'s Translation and Localisation working group — becomes a member of this delivery group in order to maintain strong connections between the T&L Working Group and this delivery group.
```

Other members would likely include a project manager employed at the University of Delhi, some named collaborators from the grant, and maybe a postdoctoral research fellow who is hired to develop and deliver trainings as described in the grant.

> The Turing Way Steering Group would not be accountable for the deliverables in the grant, nor any budget management, nor performance review or appraisals for the team members.

Reviewer: ```suggestion
*The Turing Way* Steering Group would not be accountable for the deliverables in the grant, nor any budget management, nor performance review or appraisals for the team members.
```

> These responsibilities are external to The Turing Way open source community.

Reviewer: ```suggestion
These responsibilities are external to *The Turing Way* open source community.
```

If the Steering Group agree that the proposed work is aligned to the long term strategic vision for the community they would **approve this request**.
The new delivery group would be required to set up a charter as described in <!-- CHAPTER ON DELIVERY GROUPS --> this chapter's sub-section on delivery groups.
The chair of the delivery group (Indrani) would become a member of the Steering Council.

If the Steering Group did not agree that the proposed work is aligned to the long term strategic vision for the community they would **reject this request**.
Indrani and her team could still undertake the work and continue to contribute as individual contributors.

> They can still use materials from The Turing Way under the open source licences that the project uses.

Reviewer: ```suggestion
They can still use materials from *The Turing Way under* the terms of the open source licences that the project uses.
```

Maybe a bit pedantic but the licenses we use may allow distribution as part of a new work under a different license.

Members of the Translation and Localisation Working Group would still be allowed to collaborate.

> The team members would always be expected to abide by The Turing Way code of conduct.

Reviewer: ```suggestion
The team members would always be expected to abide by *The Turing Way* code of conduct.
```
